lib/rest-adapter.js

```diff
@@ -403,6 +411,14 @@ RestAdapter.prototype._registerMethodRouteHandlers = function(router,
     // Express 4.x only supports delete
     verb = 'delete';
   }
+
+  //For now: exclude paths with parameters like :id
+  //and handler functions with err obj i.e. (err, req,res,next)
+  if (!route.path.includes(':') && handler.length < 4) {
+    handler.methodName = sharedMethod.name;
+    handler.path = route.fullPath;
+    customRouter.registerPathAndHandlers({ verb: verb, fullPath: route.fullPath }, handler);
+  }
   router[verb](route.path, handler);
```

Comment:
As I understand docs, `all` is treated the same way as other, regular verbs. If you put your route handler as the first one, it will work as you describe, but if you put it as the last one, it will work as I described it. So the task for you is to find how strong-remoting is dealing with this. IIRC there is a custom sort of routes, does it take verbs into account too?